### ISSUE TITLE
Documentation and contribution

### ISSUE THREAD

**Issue by Kilkenni** [OpenStarbound/OpenStarbound#54]
Sup. Github does not support PMs and Novaenia's profile is private, so I see no other way to contact you. I've been updating my old mod for SB recently and caught up to the news. Anyway, maybe I can be of use to this repo since I've been refreshing my memory of SB code structure for the last few days.

I'm relatively proficient with documenting via wiki (some docs on Starbounder were written or reorganized by me), so I wonder if OpenSB will use the original Starbounder wiki or it is better to have a separate one. The official wiki is administrated by Chucklefish employees (namely, Katzeus), OpenSB has no direct relation to Chucklefish, and as the project progresses it will deviate further from the "original". So I suppose keeping things separate is better.

I'm in contact with Indie Wiki Fed, so I may have some ideas about where to deploy such a wiki if people feel OpenSB needs one.

That being said, do you have any means of communication, like an IRC or a Discord? I've been wanting to fix some small issues with original SB for a while now. Nothing much, QoL fixes mostly, but some of them, I believe, would fit well into the "core game" instead of being a mod, even if it is possible to patch Lua now. I'd like to know whom should I poke if I make a pull request to get it reviewed :)

Also I wonder if there are any plans on developing the code? Any list of issues or feature requests apart from what is hosted here on Github?

Tl;dr Do you need any help?
